### PR TITLE
No ubuntu initd link

### PR DESCRIPTION
**Pull request by viliusdidit**
### Description

Ubuntu like debian doesn't like init.d -> sv symlink. This simply mimics debian.
https://github.com/chef-cookbooks/runit/issues/162

### Issues Resolved

use debian init.d template instead of symlink to sv.

### Check List

Obvious fix?

- [ ] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

### CODE REVIEW

**Comment by sauliusgrigaitis** [comment]
I can confirm that Ubuntu likes this branch a lot. Thanks!

**Comment by tas50** [comment]
@viliusdidit This looks like a good thing to get merged in. Can you rebase it against master and sign off the DCO so we can accept the contribution?

File: libraries/provider_runit_service.rb [hunk 1]
```diff
@@ -224,7 +224,7 @@ def whyrun_supported?
           end
 
           # lsb_init
-          if node['platform'] == 'debian'
+          if node['platform'] == 'debian' || node['platform'] == 'ubuntu'
```
Comment (tas50) [inline libraries/provider_runit_service.rb]: if platform_family?('debian') would work better here by also roping in any derivative distros.

Comment (viliusdidit) [reply libraries/provider_runit_service.rb]: ack. Would chef still find templates/debian/init.d.erb (source 'init.d.erb' used below) for these derivative platforms? I'm leaning for a - no, thus possibly move it into template/default/ ?

**Pull request merged**